tests/queries/tests.py

```diff
@@ -2790,6 +2804,21 @@ def test_exclude_multivalued_exists(self):
             )
         self.assertIn('exists', captured_queries[0]['sql'].lower())
 
+    def test_exclude_subquery(self):
+        subquery = JobResponsibilities.objects.filter(
+            responsibility__description='bar',
+        ) | JobResponsibilities.objects.exclude(
+            job__responsibilities__description='foo',
+        )
+        self.assertSequenceEqual(
```

Comment:
`assertCountEqual`? This sometimes fails on CockroachDB.

Reply:
True, my bad, sorry https://github.com/django/django/pull/14296